### PR TITLE
docs: copy edit getting started step 4

### PR DESCRIPTION
**Pull request by kapunahelewong**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Brings copy in line with Google guidelines. Also keeps location of `getItems()` consistent with previous step, Managing Data. In Managing Data, the cart service `getItems()` is in the `ngOnInit()`, but in the Forms step, it has moved to the constructor without instruction to do so.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

- Removes "we" per Google Developers Guide: https://developers.google.com/style/person
- Reduces future tense (for example "will be"): https://developers.google.com/style/tense
- Adds headers to code snippets
- Clears up passive voice
- Reduces parentheses: https://developers.google.com/style/parentheses

### CODE REVIEW

File: aio/content/start/forms.md [hunk 1]
```diff
@@ -21,40 +21,40 @@ First, you'll set up the checkout form model. The form model is the source of tr
       <code-example header="src/app/cart/cart.component.ts" path="getting-started/src/app/cart/cart.component.ts" region="imports">
       </code-example>
 
-      The `FormBuilder` service is provided by the `ReactiveFormsModule`, which is already defined in the `AppModule` you modified previously (in `app.module.ts`).
+      The `ReactiveFormsModule` provides the `FormBuilder` service, which `app.module.ts`, or `AppModule`, already imports.
```
Comment (jbogarthyde) [inline aio/content/start/forms.md]: I prefer parens to reduce the number of commas-- *which `AppModule` (`app.module.ts`) already imports.*

File: aio/content/start/forms.md [hunk 2]
```diff
@@ -21,40 +21,40 @@ First, you'll set up the checkout form model. The form model is the source of tr
...
 
     <code-example header="src/app/cart/cart.component.ts" path="getting-started/src/app/cart/cart.component.ts" region="checkout-form-group"></code-example>
 
-1. For the checkout process, users need to be able to submit the form data (their name and address). When the order is submitted, the form should reset and the cart should clear.
+1. For the checkout process, users need to submit their name and address. When they submit their order, the form should reset and the cart should clear.
 
-    In `cart.component.ts`, define an `onSubmit()` method to process the form. Use the `CartService#clearCart()` method to empty the cart items and reset the form after it is submitted. (In a real-world app, this method also would submit the data to an external server.)
-
-    The entire cart component is shown below:
+    1. In `cart.component.ts`, define an `onSubmit()` method to process the form. Use the `CartService` `clearCart()` method to empty the cart items and reset the form after it is submission. In a real-world app, this method also would submit the data to an external server. The entire cart component class is as follows:
```
Comment (jbogarthyde) [inline aio/content/start/forms.md]: also would -> would also

File: aio/content/start/forms.md [hunk 3]
```diff
@@ -21,40 +21,40 @@ First, you'll set up the checkout form model. The form model is the source of tr
...
     </code-example>
 
-The form model is defined in the component class. To reflect the model in the view, you'll need a checkout form.
+Now that you've defined the form model in the component class, you need
+a checkout form to reflect the model in the view.
 
 ## Create the checkout form
 
-Next, you'll add a checkout form at the bottom of the "Cart" page.
+Next, add a checkout form at the bottom of the "Cart" page.
```
Comment (jbogarthyde) [inline aio/content/start/forms.md]: I don't think you can start a new heading with "Next" -- could be "Use these steps to add..."

**Review by jbogarthyde — COMMENTED** [review]
minor edits

File: aio/content/start/forms.md [hunk 4]
```diff
@@ -21,40 +21,40 @@ First, you'll set up the checkout form model. The form model is the source of tr
       <code-example header="src/app/cart/cart.component.ts" path="getting-started/src/app/cart/cart.component.ts" region="imports">
       </code-example>
 
-      The `FormBuilder` service is provided by the `ReactiveFormsModule`, which is already defined in the `AppModule` you modified previously (in `app.module.ts`).
+      The `ReactiveFormsModule` provides the `FormBuilder` service, which `AppModule`(`app.module.ts`) already imports.
```
Comment (gkalpak) [inline aio/content/start/forms.md]: Not sure what the intention was, but I am guessing:

```suggestion
      The `ReactiveFormsModule` provides the `FormBuilder` service, which `AppModule` (`app.module.ts`) already imports.
```

...or even...

```suggestion
      The `ReactiveFormsModule` provides the `FormBuilder` service, which `AppModule` (in `app.module.ts`) already imports.
```

File: aio/content/start/forms.md [hunk 5]
```diff
@@ -21,40 +21,40 @@ First, you'll set up the checkout form model. The form model is the source of tr
...
-
-    The entire cart component is shown below:
+    1. In `cart.component.ts`, define an `onSubmit()` method to process the form. Use the `CartService` `clearCart()` method to empty the cart items and reset the form after it is submission. In a real-world app, this method would also submit the data to an external server. The entire cart component class is as follows:
 
     <code-example header="src/app/cart/cart.component.ts" path="getting-started/src/app/cart/cart.component.ts">
     </code-example>
 
-The form model is defined in the component class. To reflect the model in the view, you'll need a checkout form.
+Now that you've defined the form model in the component class, you need
+a checkout form to reflect the model in the view.
```
Comment (gkalpak) [inline aio/content/start/forms.md]: Unnecessary line break.

File: aio/content/examples/getting-started/src/app/cart/cart.component.ts [hunk 6]
```diff
@@ -24,14 +24,15 @@ export class CartComponent {
...
 // #enddocregion inject-form-builder
-    this.items = this.cartService.getItems();
-
     this.checkoutForm = this.formBuilder.group({
       name: '',
       address: ''
     });
 // #docregion inject-form-builder
   }
+  ngOnInit() {
```
Comment (gkalpak) [inline aio/content/examples/getting-started/src/app/cart/cart.component.ts]: Please, leave an empty line between constructor/methods:

```suggestion

  ngOnInit() {
```

**Review by gkalpak — APPROVED** [review]
👌

**Comment by kara** [comment]
@IgorMinar @StephenFluin Can someone review this for fw-docs-intro group approval?

**Review by StephenFluin — APPROVED** [review]

**Pull request closed without merging**